src/reducers/combined-updaters.js

```diff
@@ -182,3 +190,19 @@ export const addDataToMapUpdater = (state, {payload}) => {
 };
 
 export const addDataToMapComposed = addDataToMapUpdater;
+
+export const setFeaturesUpdater = (state, payload) => ({
+  ...state,
+  visState: visStateFeaturesUpdater(state.visState, payload),
```

Comment:
we don't need a composed updater if an action is handled in each subreducer independently. You can just add `[SET_FEATURE]` action handlers to `visState` and `uiState` reducer
```
visState = {
  ....
  [ActionTypes.SET_FEATURE]: visStateFeaturesUpdater
}

uiState = {
  ....
  [ActionTypes.SET_FEATURE]: uiStateFeaturesUpdater
}
```

The time when we need to use combined updaters is when 1 action needs to be handled in multiple reducer is a certain sequence. e.g. in `updateVisData` we use layers created by `visState` to update viewport in `mapState1`

Reply:
i initially started with independent actions and I was trying to see whether there was a performance improvement by using a composed action. But at the end i don't think it made any  different

Reply:
Done

Reply:
you are still creating `setFeaturesComposed` and `deleteFeatureComposed`.  this 2 actions don't need to be composed here. They can be handled independently inside each (visState, uiState) reducer...

Reply:
The idea behing this actions is to be used the way we use addDataToMapComposed, directly accessing kepler.gl store rather than using actions

An application can call these composed updaters when it is directly kepler.gl and want to modify the features. I am not using them at all and since i changed the logic in the second PR i can get rid of them
i am not even using those actions anywhere, i removed them